packages/rancher-sachet/charts/templates/NOTES.txt

```diff
@@ -0,0 +1,7 @@
+The Sachet app is installed into your cluster.
```

Comment:
the NOTES.txt seem to be different for each subchart. Can we make them uniform?

Reply:
fixed.